playbooks/demo_create_and_install_vios.yaml

```diff
@@ -14,6 +14,41 @@
               6365
 
   tasks:
```

Comment:
Don't Add this part in the playbook, may be once Manya or Sreenidhi's code is ready you can create one playbook to copy the image and install or upgarde vios

Reply:
Will create a playbook once install support us added